model/riscv_sys_regs.sail

```diff
@@ -882,6 +888,24 @@ function is_fiom_active() -> bool = {
     User => (menvcfg.FIOM() | senvcfg.FIOM()) == 0b1,
   }
 }
+
+bitfield Mstateen0 : bits(64) = {
+  // controls to new state introduced by the following small extensions of Priv-1.14-era:
```

Comment:
Could maybe add a link or slightly more explanation of what Priv-1.14-era means here?

Also I'm not sure if this comment makes sense 'following small extensions', but there is no following list of small extensions. Could just remove 'following small'.

Reply:
Here is a link to the specification: https://github.com/riscv/riscv-isa-manual/issues/1147. I will update the comment - it should have included Ssqosid state as the state governed by this bit for now but more state may be included by time Priv 1.14 is frozen.

Reply:
I have updated the comments.